checker/tests/interning/SymbolCompleterUsesObjectEquals.java

```diff
@@ -0,0 +1,7 @@
+import com.sun.tools.javac.code.Symbol;
```

Comment:
This won't compile without a command-line argument such as `--add-exports jdk.compiler/com.sun.tools.javac.code=ALL-UNNAMED`.  Could you remove the test or arrange for the extra command-line argument?

Reply:
Or, use a different, public class that is `@UsesObjectEquals`.  (This would require renaming the test class, too.)